change/@itwin-tree-widget-react-1c719736-b3ff-4334-ac0b-65aa410cef8e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Since this bumps a number of peer deps, should the change type be `major`? I also think the comment should mention the peer dep versions that we updated to.